packages/arb-ts/src/lib/l1Bridge.ts

```diff
@@ -75,16 +61,24 @@ export class L1Bridge {
     )
   }
 
-  public async getERC20L2Address(erc20L1Address: string) {
-    return await this.ethERC20Bridge.customL2Tokens(erc20L1Address)
+  public getERC20L2Address(erc20L1Address: string) {
+    return this.ethERC20Bridge.customL2Tokens(erc20L1Address)
```

Comment:
This seems off. Presumably it should use the calculated address if it's not custom